README.md

```diff
@@ -107,66 +114,78 @@ For more information about configuring your Cloud Pak for Data cluster, see http
 
    ![Example app](./docs/images/example-app.png)
 
+For more information on how each component can be customized and configured, check out our hosted [storybook](https://watson-developer-cloud.github.io/discovery-components)
```

Comment:
I think this should actually link here?: https://watson-developer-cloud.github.io/discovery-components/storybook. Also, it looks like the link we have at https://watson-developer-cloud.github.io/discovery-components under `Components` to our Storybook is incorrect and I think we also need to update that to include `/storybook`